### PR TITLE
prometheus-ai group added

### PR DESCRIPTION
**Pull request by benlugasi**
Hey,
I've added prometheus-ai group to operate-first/apps
This is second PR, the last one aborted.

Thanks,
Ben

### CODE REVIEW

**Review by HumairAK — APPROVED** [review]
/lgtm
/approve

**Comment by HumairAK** [comment]
Seems like you have some trailing white space: 
You can also try to run the lint tests locally: https://github.com/operate-first/apps/blob/master/contributing.md#tests 
It will attempt to auto correct the white space, at which point you can just amend your commit after staging (`git add -u`) your changes by doing `git commit --amend --no-edit -S` so you don't have to re-squash.

File: cluster-scope/base/user.openshift.io/groups/prometheus-ai/group.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+apiVersion: user.openshift.io/v1
+kind: Group
+metadata:
+    name: prometheus-ai
+users:
+    - moradnir
+    - ofir-shechtman
+    - guyelf
+    - benlugasi
```
Comment (HumairAK) [inline cluster-scope/base/user.openshift.io/groups/prometheus-ai/group.yaml:9]: new line missing

**Comment by benlugasi** [comment]
white spaces removed.
squashed to 1 commit

**Pull request merged**